moto/ec2/models/security_groups.py

```diff
@@ -520,6 +520,30 @@ def describe_security_groups(self, group_ids=None, groupnames=None, filters=None
 
         return matches
 
+    def describe_security_group_rules(self, group_ids=None, groupnames=None, filters=None):
+        matches = itertools.chain(*[x.copy().values() for x in self.groups.values()])
+        if group_ids:
+            matches = [grp for grp in matches if grp.id in group_ids]
+            if len(group_ids) > len(matches):
+                unknown_ids = set(group_ids) - set(matches)
+                raise InvalidSecurityGroupNotFoundError(unknown_ids)
+        if groupnames:
```

Comment:
We are never passing `groupnames` to this function, so that argument (and this if-block) can be removed

Reply:
Done via https://github.com/getmoto/moto/pull/5841/commits/dd324c3078725f9c1ccf33cf4a4837399aa5b3b5